README.md

```diff
@@ -1,41 +1,65 @@
 # notifications-template-preview
 
-GOV.UK Notify template preview service. Generates PNG and PDF previews of letter templates 
-created in the [GOV.UK Notify admin app](http://github.com/alphagov/notifications-admin).
+Generates PNG and PDF previews of letter templates created in the [GOV.UK Notify admin app](http://github.com/alphagov/notifications-admin).
 
-## First-time setup
+## Setting Up
 
-Since it's run in docker on PaaS, it's recommended that you use docker to run this locally.
+### `environment.sh`
 
-```shell
-  make prepare-docker-build-image
+In the root directory of the application, run:
+
+```
+echo "
+export NOTIFICATION_QUEUE_PREFIX='YOUR_OWN_PREFIX'
+"> environment.sh
 ```
 
-This will create the docker container and install the dependencies.
+Things to change:
 
-## Tests
+- Replace YOUR_OWN_PREFIX with local_dev_\<first name\>.
 
-The command to run all of the tests is
+### Docker container
+
+This app uses dependencies that are difficult to install locally. In order to make local development easy, we run app commands through a Docker container. Run the following to set this up:
 
 ```shell
-make test-with-docker
+  make bootstrap
 ```
 
-This script will run all the tests. [py.test](http://pytest.org/latest/) is used for testing.
+Because the container caches things like Python packages, you will need to run this again if you change things like "requirements.txt".
 
-Running tests will also apply syntax checking, using [flake8](https://pypi.org/project/flake8/).
+## To test the application
 
+```shell
+make test
+```
 
-## Running the Flask application
+If you need to run a specific command, such as a single test, you can use the `run_with_docker.sh` script. This is what `test-with-docker` and other `make` rules use.
```

Comment:
Hey, how do we use `run_with_docker.sh` to run a single test?

Reply:
There's an example of doing that directly below 😄.

Reply:
Ahh, yes, you are right, sorry 🤦🏼‍♀️